public/opencast-editor.svg

```diff
@@ -71,17 +73,28 @@
+   transform="translate(-166.65349,-231.21359)" /><g
+   aria-label="EDITOR"
+   id="text824"
+   style="font-weight:300;font-size:37.5px;line-height:125%;font-family:Comfortaa;-inkscape-font-specification:'Comfortaa Light';letter-spacing:0px;word-spacing:0px;stroke-width:0.9375px"><path
+     d="m 425.88736,77.535515 h 18.0375 c 0.6,0 1.125,-0.4875 1.125,-1.125 0,-0.6 -0.525,-1.125 -1.125,-1.125 h -16.9125 v -11.25 h 12.4125 c 0.6375,0 1.125,-0.525 1.125,-1.1625 0,-0.6375 -0.4875,-1.125 -1.125,-1.125 h -12.4125 v -11.25 h 16.9125 c 0.6,0 1.125,-0.525 1.125,-1.125 0,-0.6375 -0.525,-1.125 -1.125,-1.125 h -18.0375 c -0.6375,0 -1.125,0.4875 -1.125,1.125 v 27.0375 c 0,0.6375 0.4875,1.125 1.125,1.125 z"
+     style="fill:#ffffff"
+     id="path176" /><path
+     d="m 451.61239,77.535515 h 6.75 c 8.5875,0 14.6625,-6.075 14.6625,-14.6625 0,-8.55 -6.075,-14.625 -14.6625,-14.625 h -6.75 c -0.675,0 -1.125,0.45 -1.125,1.125 v 27.0375 c 0,0.675 0.45,1.125 1.125,1.125 z m 1.125,-2.025 v -25.2375 h 5.625 c 7.275,0 12.4125,5.3625 12.4125,12.6 0,7.275 -5.1375,12.6375 -12.4125,12.6375 z"
+     style="fill:#ffffff"
+     id="path178" /><path
+     d="m 480.14988,77.535515 c 0.6375,0 1.125,-0.4875 1.125,-1.125 v -27.0375 c 0,-0.6375 -0.4875,-1.125 -1.125,-1.125 -0.6375,0 -1.125,0.4875 -1.125,1.125 v 27.0375 c 0,0.6375 0.4875,1.125 1.125,1.125 z"
+     style="fill:#ffffff"
+     id="path180" /><path
+     d="m 507.82489,48.248015 h -21.225 c -0.6,0 -1.0125,0.4125 -1.0125,1.0125 0,0.6 0.4125,1.0125 1.0125,1.0125 h 9.4875 v 26.1375 c 0,0.6375 0.4875,1.125 1.125,1.125 0.6375,0 1.125,-0.4875 1.125,-1.125 v -26.1375 h 9.4875 c 0.6,0 1.0125,-0.4125 1.0125,-1.0125 0,-0.6 -0.4125,-1.0125 -1.0125,-1.0125 z"
+     style="fill:#ffffff"
+     id="path182" /><path
+     d="m 524.8124,77.910515 c 8.55,0 15,-6.45 15,-15.0375 0,-8.55 -6.45,-15 -15,-15 -8.5875,0 -15.0375,6.45 -15.0375,15 0,8.5875 6.45,15.0375 15.0375,15.0375 z m 0,-2.25 c -7.275,0 -12.7875,-5.5125 -12.7875,-12.7875 0,-7.2375 5.5125,-12.75 12.7875,-12.75 7.2375,0 12.75,5.5125 12.75,12.75 0,7.275 -5.5125,12.7875 -12.75,12.7875 z"
+     style="fill:#ffffff"
+     id="path184" /><path
+     d="m 563.62489,75.810515 -6.45,-10.3125 c 3.7875,-1.0875 6.3,-4.2375 6.3,-8.3625 0,-5.2125 -3.9,-8.8875 -9.375,-8.8875 h -7.1625 c -0.675,0 -1.125,0.45 -1.125,1.125 v 27.0375 c 0,0.675 0.45,1.125 1.125,1.125 0.675,0 1.125,-0.45 1.125,-1.125 v -10.5 h 6.0375 0.675 l 6.9,11.025 c 0.225,0.3375 0.45,0.6 1.0125,0.6 1.1625,0 1.575,-0.7125 0.9375,-1.725 z m -15.5625,-12.15 v -13.1625 h 6.0375 c 4.1625,0 7.125,2.7375 7.125,6.6375 0,3.825 -2.9625,6.525 -7.125,6.525 z"
+     style="fill:#ffffff"
+     id="path186" /></g>
 <text
    xml:space="preserve"
    style="font-size:37.5px;line-height:1.25;font-family:'DejaVu Sans Mono';-inkscape-font-specification:'DejaVu Sans Mono';letter-spacing:0px;word-spacing:0px;stroke-width:0.9375"
```

Comment:
Starting here, there is some weird out-of-bounds text box with the text "q". That should be removed as well.

Reply:
Since you started complaining about unnecessary stuff in the SVG, I removed this and a buch of additional data Inkscape threw in there we don't need.

You can probably remove more if you want, but I think it's fine like this.